scripting/get5/stats.sp

```diff
@@ -227,8 +227,13 @@ public Action Stats_PlayerDeathEvent(Event event, const char[] name, bool dontBr
       IncrementPlayerStat(attacker, STAT_KILLS);
       g_PlayerRoundKillOrAssistOrTradedDeath[attacker] = true;
 
-      if (headshot)
+      if (headshot) {
         IncrementPlayerStat(attacker, STAT_HEADSHOT_KILLS);
+      }
+
+      if (StrEqual(weapon, "knife") || StrEqual(weapon, "knife_t")) {
```

Comment:
The force-pushed was to add `knife_t`, as I found out during testing that the knife has a different value on the T side. GG Valve.